app/controllers/categories_controller.rb

```diff
@@ -2,11 +2,20 @@ class CategoriesController < ApplicationController
 
   def show
     @category = Category.find_by!(slug: params[:slug])
-    @articles = @category.articles.page(params[:page]).per(15)
+    @articles = @category.articles.live.published.page(params[:page]).per(15)
```

Comment:
Good catch adding the `.live.published` to these collections. We don't currently have any unpublished drafts. But this would've allowed them through. 🌟